src/ledger/LedgerManagerImpl.cpp

```diff
@@ -212,7 +212,7 @@ LedgerManagerImpl::startNewLedger()
         ledger.maxTxSetSize = cfg.DESIRED_MAX_TX_PER_LEDGER;
     }
 
-    startNewLedger(ledger);
+    startNewLedger(std::move(ledger));
```

Comment:
I think it's clearer to just have `startNewLedger` take a const reference: the only reason to pass around by value is if the function is going to make a copy anyways for some reason, which is not the case here.

Reply:
nm - didn't see your other comment on the same issue that ledgerDelta requires a non const